overrides/config-overrides/normal/betterquesting/DefaultQuests.json

```diff
@@ -58323,7 +58323,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "§2This tab was added to help you visualise and give you tips on making chemicals, among other important things to automate. Use it wisely!§r\n\nEach quest in the tab will give you a description of the machine and the minimum voltage required to run the recipe. Furthermore, although each quest can be completed via a checkbox, all quests in a processing line will auto-complete once you get the item positioned on the right. Either the checkbox or achieving the product will complete the quests.\n\n§oYou may have to hold the item for a few seconds for the whole line to auto-complete.\n\nThis is not complete, and is published out for testing purposes. Currently, only Pulsating Polymer Clay and Cobbleworks are implemented, but more chains are to come. These are Polyethylene and Polyvinyl Chloride, Oil Processing, Epoxy Resin, Rocket Fuel, Polytetrafluoroethylene, High-Octane Gasoline, Cetane-Boosted Diesel, Polybenzimidazole, Naquadah Processing and Platinum Group Processing. If you have any suggestions on other chains or on how to improve this tab, feel free to DM Integer Limit#0458 on Discord or create an issue on GitHub.",
+          "desc:8": "§2This tab was added to help you visualize and give you tips on making chemicals, among other important things to automate. Use it wisely!§r\n\nEach quest in the tab will give you a description of the machine and the minimum voltage required to run the recipe. Furthermore, although each quest can be completed via a checkbox, all quests in a processing line will auto-complete once you get the item positioned on the right. Either the checkbox or achieving the product will complete the quests.\n\n§oYou may have to hold the item for a few seconds for the whole line to auto-complete.\n\nThis is not complete, and is published out for testing purposes. Currently, only Pulsating Polymer Clay and Cobbleworks are implemented, but more chains are to come. These are Polyethylene and Polyvinyl Chloride, Oil Processing, Epoxy Resin, Rocket Fuel, Polytetrafluoroethylene, High-Octane Gasoline, Cetane-Boosted Diesel, Polybenzimidazole, Naquadah Processing and Platinum Group Processing. If you have any suggestions on other chains or on how to improve this tab, feel free to DM Integer Limit#0458 on Discord or create an issue on GitHub.",
```

Comment:
Can you change this to message the development thread, or just remove the discord part of it?